v3/makefile

```diff
@@ -14,7 +14,7 @@ CMDS = zlint zlint-gtld-update
 CMD_PREFIX = ./cmd/
 BUILD = $(GO_ENV) go build
 TEST = $(GO_ENV) GORACE=halt_on_error=1 go test -race
-INT_TEST = $(GO_ENV) go test -v -tags integration -timeout 20m ./integration/... -parallelism $(PARALLELISM) $(INT_FLAGS)
+INT_TEST = $(GO_ENV) go test -v -tags integration -timeout 20m ./integration/. -parallelism $(PARALLELISM) $(INT_FLAGS)
```

Comment:
Was `...` not working here?

Reply:
It was trying to run the integration tests on this directory recursively which led which led to these flags getting propagated down into the plain Go unit tests which did not understand, in particular, `-parallelism`. I can take a second look at this, but I _think_ these flags are custom to the integration tests so they won't be understood by them.